frontend/src/pages/login/forgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { TextField, Button, Snackbar } from '@material-ui/core';
+import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
```

Comment:
Since this is from the material ui lab, we should test in multiple web browsers and in different screen resolution

Reply:
will look at tests

Reply:
cant test, but looks good on my machine :)